elixir/apps/domain/lib/domain/release.ex

```diff
@@ -5,11 +5,18 @@ defmodule Domain.Release do
   @repos Application.compile_env!(@otp_app, :ecto_repos)
 
   def migrate(opts \\ []) do
+    IO.puts("Starting sentry app..")
```

Comment:
Was this meant to be a `Logger.info` call?

Reply:
I'll just use Logger

Reply:
Ah, ok, didn't think about Logger not being setup at this point.  If this is the convention then I say leave it.